### PR TITLE
Add FieldInfoMirror.FullName

### PR DESCRIPTION
**Pull request by jbevain**

### CODE REVIEW

**Comment by jbevain** [comment]
@vargaz this one's for you as well.

File: mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs [hunk 1]
```diff
@@ -200,6 +200,20 @@ public Boolean IsNotSerialized
 					res.Add (attr);
 			return res.ToArray ();
 		}
+
+		public string FullName {
+			get {
```
Comment (vargaz) [inline mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs]: This is ok, but the commit message says ToString ().

Comment (jbevain) [inline mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs]: Good catch, I'll reword the commit. I'm basically making it symmetric with MethodInfoMirror.

File: mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs [hunk 2]
```diff
@@ -200,6 +200,20 @@ public Boolean IsNotSerialized
...
+				string type_namespace = DeclaringType.Namespace;
+				string type_name = DeclaringType.Name;
+				StringBuilder sb = new StringBuilder ();
+				if (type_namespace != String.Empty)
+					sb.Append (type_namespace + ".");
```
Comment (Therzok) [inline mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs]: mono/debugger-libs uses a `.Append (type_namespace).Append('.')` here.

File: mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs (hunk 1 above)
Comment (jbevain) [inline mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs]: Fixed the commit message.

File: mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs (hunk 2 above)
Comment (jbevain) [inline mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs]: Fixed.

**Pull request merged**